src/coreComponents/finiteVolume/SurfaceElementStencil.hpp

```diff
@@ -433,6 +470,101 @@ inline void SurfaceElementStencilWrapper::computeWeights( localIndex iconn,
   }
 }
 
+GEOSX_HOST_DEVICE
+inline void SurfaceElementStencilWrapper::computeWeights( localIndex iconn,
+                                                          CoefficientAccessor< arrayView3d< real64 const > > const & coefficient,
+                                                          CoefficientAccessor< arrayView3d< real64 const > > const & coefficientMultiplier,
+                                                          R1Tensor const & gravityVector,
+                                                          real64 (& weight)[MAX_NUM_OF_CONNECTIONS][2] ) const
+{
+  // TODO: this should become star-delta method
+  real64 sumOfTrans = 0.0;
+  for( localIndex k=0; k<numPointsInFlux( iconn ); ++k )
+  {
+    localIndex const er  =  m_elementRegionIndices[iconn][k];
+    localIndex const esr =  m_elementSubRegionIndices[iconn][k];
+    localIndex const ei  =  m_elementIndices[iconn][k];
+
+    real64 const mult = ( LvArray::math::abs( LvArray::tensorOps::AiBi< 3 >( m_cellCenterToEdgeCenters[iconn][k], gravityVector ) ) > 1e-10 )
```

Comment:
This limit of 1e-10 is used several times in this class. Do you want to define a constant for it?

Reply:
Done